source/_integrations/matter.markdown

```diff
@@ -46,13 +46,23 @@ Unlike other common radio-based protocols for IoT, (like Zigbee, Z-Wave, and Blu
 
 Home Assistant is a so-called "_controller_" in a Matter ecosystem, meaning that it can control Matter-based devices. Other examples of Matter controllers are the Google Nest products, Apple HomePod speakers, Samsung SmartThings Station, and some newer Amazon Echo devices.
 
-## Thread
+## What does Thread have to do with Matter?
 
-Matter goes hand-in-hand with (but is not the same as) {% term Thread %}. {% term Thread %} is a low power radio mesh networking technology. Much like Zigbee, but with the key difference that it is _IP-addressable_, making it the perfect companion transport protocol for Matter.
+{% term Thread %} is a low power radio mesh networking technology. Much like Zigbee, but with the key difference that it is _IP-addressable_, making it the perfect companion transport protocol for Matter. 
 
-<div class='note'>
-Many devices that (will) hit the market will use {% term Thread %} for radio communication and Matter as a control protocol, but this is not guaranteed. For example, Thread-based devices are available that only support Apple HomeKit or some vendor-specific communication protocol. There are also a few cases where you need to apply for a (beta) firmware update on the device to enable Matter as a communication protocol. Therefore, do not assume Matter support when you see a Thread logo when looking for devices. Please be sure to look for the *Matter* logo itself (on either Wi-Fi/Ethernet-based devices or {% term Thread %}) or any other confirmation by the manufacturer that the device supports Matter.
-</div>
+The figure below illustrates the landscape of {% term Matter %}, {% term Thread %}, and Border routers. Instead of Matter, you could also see another protocol here, such as HomeKit. The Thread border router forwards traffic from the devices in the Thread network to devices outside that network.
```

Comment:
I think in this context I'd leave out HomeKit. We will inform users below again that Thread doesn't necessarily mean that a device is also using the Matter smart home protocol.

```suggestion
The figure below illustrates the landscape of {% term Matter %}, {% term Thread %}, and Border routers. The Thread border router forwards traffic from the devices in the Thread network to devices outside that network.
```